### PR TITLE
Boring dependency upgrade, Jekyll 3.4.x

### PR DESCRIPTION
**Pull request by wjdp**
Periodic upgrade of packages. Mainly to get us up-to-date with Jekyll 3.4.x

### CODE REVIEW

**Comment by wjdp** [comment]
@davidtaylorhq Since you've got your head in this area, mind checking over?

**Comment by nhyty** [comment]
@wjdp, you're decision. How long will it take to fix the docker/vagrant stuff @davidtaylorhq

**Comment by davidtaylorhq** [comment]
About 5 minutes, when I get around to it

File: .travis.yml [hunk 1]
```diff
@@ -10,10 +10,11 @@ cache:
   - tmp/
   - ".sass-cache"
 before_install:
-- nvm install 4.2.6
+- nvm install 6.10.0
 install:
 - bundle install --jobs=3 --retry=3 --deployment --path=${BUNDLE_PATH:-vendor/bundle}
 - npm install
+- npm rebuild node-sass # Remove this soon
```
Comment (davidtaylorhq) [inline .travis.yml]: @wjdp what's this line for?

Comment (wjdp) [inline .travis.yml]: When you install node-sass you build a binary. We cache this binary. When you update node that binary needs rebuilding. :persevere:

Comment (davidtaylorhq) [inline .travis.yml]: Cool, so not an issue for the Docker setup (no cache)

Comment (wjdp) [inline .travis.yml]: Nope , shouldn't be an issue for you

**Pull request merged**